tests/e2e/support/api/share/share.ts

```diff
@@ -41,3 +43,48 @@ export const createShare = async ({
   })
   checkResponseStatus(response, 'Failed while creating share')
 }
+
+export const acceptShare = async ({ user, path }: { user: User; path: string }): Promise<void> => {
+  const shareId = await getShareIdFromShareWithMePage({ user, path })
+  console.log(shareId)
+  const response = await request({
+    method: 'POST',
+    path: join('ocs', 'v2.php', 'apps', 'files_sharing', 'api', 'v1', 'shares', 'pending', shareId),
+    user: user
+  })
+  checkResponseStatus(response, 'Failed while accepting share')
+}
+
+export const getShareIdFromShareWithMePage = async ({
```

Comment:
```suggestion
export const getShareId = async ({
```

Reply:
@saw-jan The query params used in endpoints to get `shareId` only work for `shareWithMePage` so that I used `getShareIdFromShareWithMePage` name. If we use `getShareId` name, we can refactor the method now to work for other cases or we can refactor it whenever needed for other cases. What will you say?